tests/constants/admincli/vmdkops.go

```diff
@@ -31,4 +31,10 @@ const (
 
 	// StartService starts vmdkops service
 	StartService = vmdkopsd + "start"
+
+	// DBNotConfigured - DB mode not configured
+	DBNotConfigured = "NotConfigured"
```

Comment:
This will be used by cross-esx tests when we start automating them. I will keep it here.